CRM/Event/BAO/Event.php

```diff
@@ -1172,6 +1172,7 @@ public static function sendMail($contactID, $values, $participantId, $isTest = F
           'customPost' => $profilePost[0],
           'customPost_grouptitle' => $customPostTitles,
           'participantID' => $participantId,
+          'contactID' => $contactID,
```

Comment:
Ug you have pulled in some unrelated stuff

Reply:
test all passed now. Are you able to merge this?